### DISCUSSION TITLE
Need professional Infos: Shortest update time for setting limits

### DISCUSSION THREAD

**Discussion by 2000ede** [tbnobody/OpenDTU#1723] [Q&A]
Hello,
i need you help!

As soon as I send the openDTU limit, it doesn't exactly perform as well. It expresses itself like this:
- API calls sometimes take longer than 10 seconds or no response
- New limits lead to long waiting times within the DTU.

If I send the limit every 6 seconds, it still works just fine.
At every 3 seconds the openDTU can no longer do it.

Question: What is a normal update time?
Can I still optimize something?

Do you need further information? I would be happy to post them here.

![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/0497446f-06cd-49c9-94e6-7c5c0fd7f1d9)

**Comment by DPO99** [comment]
Normalerweise reicht es alle 60 Sekunden. Es ist total unnötig alle 6 Sekunden etwas zu senden....leb damit, dass du niemals 100% erreichen wirst und stress die DTU nicht so ;).

**Reply by 2000ede** [reply]
Da will ich dir nicht widersprechen. Trotzdem möchte ich schon wissen wo die Grenzen sind. Um so genauer läuft es und um so mehr verstehe ich auch.

**Reply by DPO99** [reply]
Dann schau mal hier, da würde bisschen drüber gesprochen. #1240
